src/N98/Util/Console/Helper/DatabaseHelper.php

```diff
@@ -51,11 +66,11 @@ public function detectDbSettings(OutputInterface $output)
             return;
         }
 
-        if (!isset($config['db']['connection']['default'])) {
-            throw new RuntimeException('Cannot find default connection config in app/etc/config.php');
+        if (!isset($config['db']['connection'][$this->_connectionType])) {
+            throw new RuntimeException(sprintf('Cannot find "%s" connection config in app/etc/config.php', $this->_connectionType));
```

Comment:
I think this is now env.php with Magerun2 (existing mistake), we should fix it along the lines.